docs/config/@config.md

```diff
@@ -0,0 +1,197 @@
+`@Config`
+=======
+
+The `@Config` annotation is an alternative to `Configuration`. 
+
+Table of Contents
+-------------
+
+* [Basics][basics]
+* [@Config Use][configuse]
+* [@Comment Use][commentuse]
+* [@Name Use][nameuse]
+* [@RangeInt Use][rangeintuse]
+* [@RangeDouble Use][rangedoubleuse]
+* [@LangKey Use][langkeyuse]
+* [@RequiresMcRestart Use][requiresmcrestartuse]
+* [@RequiresWorldRestart Use][requiresworldrestartuse]
+
+Basics
+------
+
+A Config should contain static fields, one per config option. There are a few exceptions, such as [Nested Configs][nestedconfigs] and [Sub Categories][subcategories]. Note that you can use an enum as a config field. To add information to and control config values, you can use the plethora of annotations provided in the `@Config` class.
```

Comment:
What are you defining as the difference between a "Nested Config" and a "Sub Category". You appear to be doing the same thing in both.